test/doctests.jl

```diff
@@ -1,9 +1,15 @@
 using Test, Documenter, COBREXA
 
 ex = quote
-    using COBREXA, Tulip
+    using COBREXA, Tulip, Downloads
     include(joinpath(dirname(pathof(COBREXA)), "..", "test", "data", "test_models.jl"))
     model = test_LP()
+    core_model_path =
+        joinpath(dirname(pathof(COBREXA)), "..", "test", "data", "e_coli_core.json")
+    Downloads.download(
```

Comment:
use the download function from tests, ideally declare this variable in test_models.jl (or have doctest_models.jl) so that this code stays clean.

Reply:
So I removed the doctest for the model joining function because it is difficult to test properly with the pretty printing, I have however added the files as you suggested because in a future PR we will add doctests for other functions that rely on the models being available.